README.md

```diff
@@ -12,15 +12,19 @@
 
 You should now have 100 fake eth! You're now fake rich.
 
-## Getting Started (Frontend)
+## Frontend Setup
 
 1. `npm install`
-1. `npm run contract:prepare` (this builds your contracts)
-1. `npm run contract:deploy` (this deploys your contracts to your local blockchain)
+2. Create a new file named `.env` in the 'frontend' directory alongside `.env.mainnet` and `.env.testnet` [**Edit this file to include the following lines of code**](https://github.com/CryptoBlades/cryptoblades/blob/main/frontend/.env.testnet#L1-L5)
+3. `npm run contract:deploy` (this deploys your contracts to your local blockchain)
+4. `npm run start:frontend`
 
-You will also need to create a `.env` file alongside `.env.mainnet` and `.env.testnet`. Inside this file you [**should only copy the feature flags from either other file**](https://github.com/CryptoBlades/cryptoblades/blob/main/frontend/.env.testnet#L1-L5), and the **file must be called `.env`**. All addresses and other variables are set correctly for local development. **If you don't create a `.env` file, you're going to be missing a lot of the UI.**
-
-But if not specified, the deployed contracts will be pulled in automatically.
+For Windows developers experiencing errors follow these steps:
+1. `Remove-item build`
```

Comment:
this seems like it's specific for powershell, I'd like to see this changed to something more agnostic (if not just `rm -rf`) because some folks don't recognize powershell at first sight - same as the #4. 

also, when making numbered lists, do `1.` in front of all of them - this means items can move around without having to rearrange the entire list - there should be examples in this doc

Reply:
Got it!